test/browser.transport.test.js

```diff
@@ -148,31 +148,31 @@ describe('post', function () {
 
     var uuid = 'd4c7acef55bf4c9ea95e4fe9428a8287';
 
-    it('should use fetch when requested', function (done) {
+    it.only('should use fetch when requested', function (done) {
```

Comment:
Is this here intentionally?

Reply:
[whoops](https://www.youtube.com/watch?v=wuI65gEinLc&pp=ygUPd2hvb3BzIHdob29wc2ll)